.github/workflows/publish.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish to PyPI
+on: workflow_dispatch  # manual invocation
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.10'
+      # Generate the source and binary distributions with setup.py
+      # https://packaging.python.org/tutorials/packaging-projects/#generating-distribution-archives
+      - run: python3 -m pip install --upgrade build
+      - run: python3 -m build
+      # Upload build as workflow artifact in case we need it
+      - uses: actions/upload-artifact@v3
+        with:
+          name: dist
+          path: dist/
+      # Upload the build to PyPI using a GitHub Action provided by PyPA
+      - uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_API_TOKEN }}
+          repository_url: https://pypi.org/legacy/
```

Comment:
Instead of having two separate workflows to maintain, can we use a single one that's parameterized with [an input](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs) (boolean or select a value) to choose between `pypi.org` and `test.pypi.org`?

Reply:
Done: 8bbbe1c